langkit/__init__.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+__default_data_folder = "langkit_data"
+
+
+def get_data_home() -> str:
```

Comment:
I had to move `get_data_home` to here because it was creating the data folder under the `metrics` folder, instead of the root. I'm not sure this is the best way to solve this, though